src/test/config.sol

```diff
@@ -1787,10 +1787,10 @@ contract Config {
             lerp:         false
         });
         afterSpell.collaterals["RETH-A"] = CollateralValues({
-            aL_enabled:   false,
-            aL_line:      0,
-            aL_gap:       0,
-            aL_ttl:       0,
+            aL_enabled:   true,
+            aL_line:      5 * MILLION,
+            aL_gap:       3 * MILLION,
+            aL_ttl:       8 hours,
```

Comment:
Consider also including test coverage for collateral oboarding integration as the ilk will be enabled with the spell, to ensure correct beahaviour, see #124